src/js/modules/forms/components/ArticleForm.js

```diff
@@ -1,16 +1,15 @@
-import React, { Component }        from "react";
-import TitleInput                  from "./ArticleInputs/TitleInput";
-import ContentInput                from "./ArticleInputs/ContentInput";
-import SectionInput                from "./ArticleInputs/SectionInput";
-import ContributorsList            from './ContributorsList'
-import ContributorsInput           from "./ArticleInputs/ContributorsInput";
-import injectSheet                 from 'react-jss'
-import { createArticle }           from './../actions'
-import { connect }                 from 'react-redux'
-import Paper                       from 'material-ui/Paper'
-import { saveArticleData }         from './../actions'
-import RaisedButton                from 'material-ui/RaisedButton'
-import { randomArticleSelector }   from '../../articles/selectors'
+import React, { Component } from "react";
+import TitleInput from "./ArticleInputs/TitleInput";
+import ContentInput from "./ArticleInputs/ContentInput";
+import SectionInput from "./ArticleInputs/SectionInput";
+import ContributorsList from "./ContributorsList";
+import ContributorsInput from "./ArticleInputs/ContributorsInput";
+import injectSheet from "react-jss";
+import { connect } from "react-redux";
+import Paper from "material-ui/Paper";
+import { saveArticleData, createArticle } from "./../actions";
+import RaisedButton from "material-ui/RaisedButton";
+import { randomArticleSelector } from "../../articles/selectors";
 
```

Comment:
import articles modules, then in the code do `articles.selectors.randomArticleSelector(state)`